client/src/components/NewsWidget/newsAPI.js

```diff
@@ -0,0 +1,12 @@
+const NEWS_API_KEY = "91e24cff34754f14aae4b9eaf858855d";
```

Comment:
We should be storing API keys in a .env file, rather than in the code.

Reply:
I raised this point in the team meeting on Saturday and the consensus was to just leave it as is. Is it critical that we do it the way you suggest for the purpose of assignment 2 ?